lib/puppet/provider/package/brewcommon.rb

```diff
@@ -22,7 +22,7 @@ def self.execute(cmd)
 
     warn('Homebrew will be dropping support for root-owned homebrew by November 2016. Though this module will not prevent you from running homebrew as root, you may run into unexpected issues. Please migrate your installation to a user account -- this module will enforce this once homebrew has officially dropped support for root-owned installations.') if owner == 0
 
-    super(cmd, :uid => owner, :gid => group, :combine => true,
+    super(cmd, :uid => owner, :gid => group, :combine => false,
```

Comment:
This will discard all `stderr` output from brew. Are we sure none of our output parsing matches on stderr? Note to self: check this.